scripts/xtensa-build-all.sh

```diff
@@ -322,9 +379,11 @@ do
 		make overrideconfig
 	fi
 
-	if [ 'tgl' != "${platform}" ]; then
+	# TGL needs MEU arguments for signing
+	if [ 'tgl' == "${platform}" ] && [ -n "${MEU_PATH_OPTION}" ] \
+	   && [ -n "${MEU_OPENSSL_OPTION}" ]; then
 		make bin -j "${BUILD_JOBS}" ${BUILD_VERBOSE}
-	else # FIXME: finish gcc support for tgl
+	else
```

Comment:
`make bin` signs and `make sof` does not so the test doesn't seem correct to me. Travis logs could be useful.

Reply:
not sure what you mean here. can you please elaborate?

Reply:
The if/then/else logic seems wrong and I think that's why it broke the build, see https://travis-ci.org/github/thesofproject/sof/builds/727509694

`make sof` builds the firmware but stops before signing it.
`make bin` invokes `make sof` and then it signs the firmware.